### DISCUSSION TITLE
Consider a circle with radius 10 units. What is the exact length of an arc that subtends a central angle of 45 degrees ?

### DISCUSSION THREAD

**Discussion by Sriansh-raj** [Sriansh-raj/GitHub-Achievements#29] [Galaxy Qs]
Consider a circle with radius 10 units. What is the exact length of an arc that subtends a central angle of 45 degrees ?

**Comment by Mayar-Abuzahra** [comment]


**Reply by Sriansh-raj** [reply]
Congratulations @Mayar-Abuzahra you just earned a prestigious `Galaxy Brain` achievement.
🎉

**Reply by Sriansh-raj** [reply]
@Mayar-Abuzahra now pls give some time and make a pr and enter your name in [Users_who_got_Galaxy_Brain.md](https://github.com/Sriansh-raj/GitHub-Achievements/blob/main/Users_who_got_Galaxy_Brain.md) to make a history 😄
